### ISSUE TITLE
"What is GDExtension?" Fails to list supported platforms

### ISSUE THREAD

**Issue by rshouker** [godotengine/godot-docs#8544]
**Your Godot version:**
4.2
**Issue description:**
The documentation doesn't specify on which platforms a GDExtension be targeted. I understand that that web target was added in v4.2, but looking at the doc for v4.1, there is no mention there that web was not supported.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/tutorials/scripting/gdextension/what_is_gdextension.html

**Comment by tetrapod00** [comment]
Generally, it's now past the point where we should be making changes to the 4.0 and 4.1 docs. So if GDExtension now supports all platforms, no further note is needed, and this issue can be closed. (If anyone did submit a PR changing the 4.0 and 4.1 docs, we would probably accept it though.)

It may still be worth explicitly mentioning that GDExtension supports all platforms in the page somewhere.

@dsnopek Can you confirm that GDExtension is indeed supported on all platforms? If so, we can either close this issue with no changes or maybe add a short explicit note to resolve it.

**Comment by dsnopek** [comment]
Yeah, at this point GDExtension supports all the platforms that Godot supports

**Comment by skyace65** [comment]
Closing this per dsnopek's comment.
